awx/__init__.py

```diff
@@ -106,46 +89,13 @@ def prepare_env():
 
     if not settings.DEBUG:  # pragma: no cover
         warnings.simplefilter('ignore', DeprecationWarning)
-    # Monkeypatch Django find_commands to also work with .pyc files.
```

Comment:
This is surely a relic of closed source `tower` shipping without source `.py` files